### PR TITLE
Add option to exclude tags

### PR DESCRIPTION
**Pull request by zalsader**
The current code for `lookup_latest_tag` finds the latest tag, whether it was generated by helm or not. This option allows excluding a certain pattern of tags. e.g. `v*` to exclude version tags.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -32,6 +32,9 @@ inputs:
     description: Mark the created GitHub release as 'latest'
     required: false
     default: true
+  exclude_tags:
+    description: "Exclude tags matching a this glob when looking for latest release"
```
Comment (cpanato) [inline action.yml:36]: can you rephrase this sentence, I feel that is not clear enough

**Review by cpanato — CHANGES_REQUESTED** [review]
thanks!

can you also add this option to the readme?

**Comment by zalsader** [comment]
Added

**Comment by zalsader** [comment]
@cpanato could you please take a look again?

**Comment by cpanato** [comment]
thanks. Before we merge this, is it possible to show test evidence that it is working? also show the a test that the normal case still works 

thanks

**Comment by zalsader** [comment]
Closing because I no longer have access to the original repository.
